WebRoot/js/zimbraMail/share/controller/ZmListController.js

```diff
@@ -352,9 +352,9 @@ function(view) {
 
 	ZmBaseController.prototype._initializeTabGroup.apply(this, arguments);
 
-	var navToolBar = this._navToolBar[view];
```

Comment:
Notes: We are considering navToolbar as part of main toolbar element. hence we are removing navToolbar from Tab navigation. User can now navigate to navtoolbar element with keyboard Left/Right arrow key when focus is on Main toolbar element